packages/snowpack/src/build/build-pipeline.ts

```diff
@@ -50,7 +50,7 @@ async function runPipelineLoadStep(
         messageBus.emit(msg, {
           ...data,
           id: step.name,
-          msg: data.msg && `[${srcPath}] ${data.msg}`,
+          msg: data.msg && `${data.msg} (${srcPath})`,
```

Comment:
- can we make srcPath relative to the cwd?
- personal preference for `[]` over `()` for technical info in logs